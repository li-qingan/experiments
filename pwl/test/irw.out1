PIN:MEMLATENCIES 1.0. 0x0
#
# IRW stats
#
Total Instruction:	11703
Total Reads:	3209
Total Writes:	1889
Total fast-clock:	20579
Total footprint in Byte:	3504
