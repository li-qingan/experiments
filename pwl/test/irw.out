PIN:MEMLATENCIES 1.0. 0x0
#
# IRW stats
#
Total Instruction:	11663
Total Reads:	3209
Total Writes:	1878
Total fast-clock:	20506
Total footprint in Byte:	3504
